app/src/main/java/org/simple/clinic/registration/RegistrationModule.kt

```diff
@@ -13,12 +12,13 @@ import org.threeten.bp.Duration
 class RegistrationModule {
 
   @Provides
-  fun config(): Single<RegistrationConfig> {
-    return Single.just(RegistrationConfig(
```

Comment:
If you look at `RegistrationFacilitySelectionController`, this class was being injected as a `Single<RegistrationConfig>`. Reactive constructs like `Observable` and `Single` add more complexity and we should only use them if they are necessary.

This was not necessary and we changed it to just directly the config instance directly. Since that class was the only one which was using this dependency, we could remove it from the module.